### DISCUSSION TITLE
Markdown keyboard shortcut for adding assets

### DISCUSSION THREAD

**Discussion by lorisleiva** [statamic/cms#4813] [Q&A] [answered]
Hey guys 👋 

Is there a keyboard shortcut on the Markdown editor to add a new asset? Similarly to how we can press `cmd+k` for adding a link.

At the moment, every time I need to add an image to one of my articles I've got to scroll all the way up and click the image icon which is pretty time-consuming in the long run.

**Comment by edalzell** [comment, marked as answer]
This would make a nice FR: https://github.com/statamic/ideas/issues/new/choose

**Reply by lorisleiva** [reply]
Done https://github.com/statamic/ideas/issues/703
